package/wpe/wpeframework-plugins/wpeframework-plugins.mk

```diff
@@ -60,6 +60,11 @@ ifeq ($(BR2_PACKAGE_WPEFRAMEWORK_POWER),y)
 WPEFRAMEWORK_PLUGINS_CONF_OPTS += -DWPEFRAMEWORK_PLUGIN_POWER=ON
 endif
 
+ifeq ($(BR2_PACKAGE_WPEFRAMEWORK_COBALTBROWSER),y)
```

Comment:
Can we rename this to WPEFRAMEWORK_COBALT instead of COBALTBROWSER? It isn't technically a full scale browser so lets avoid confusion.

Reply:
Thank you @wouter-lucas for the comments.
I will update it